installation/linux.md

```diff
@@ -7,7 +7,27 @@ nav_order: 3
 
 # Linux Installation Instructions
 
-## Dependencies
+As of version 0.3.1, an URBANopt installer (Ubuntu 18.04) is available to install the URBANopt CLI, Ruby 2.5.x, and OpenStudio SDK at the same time.  If you'd rather install the dependencies manually, view the [manual install](#manual-install) section below.
```

Comment:
@kflemin Shouldn't this be CLI version 0.4.0? And we haven't merged that into develop on the CLI yet.

Reply:
we already have installers made for 0.3.1